README.md

```diff
@@ -23,25 +23,129 @@ import {
+
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      dataSource: new ListView.DataSource({
+        rowHasChanged: this._rowHasChanged.bind(this),
+      }),
+    };
+  }
+
+  async componentWillMount() {
+    // Initial fetch for data, assuming that listData is not yet populated.
+    this._loadMoreContentAsync();
+  }
+
+  componentDidMount() {
+    // Trigger a re-render when receiving initial props.
+    this.updateDataSource(this.props);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    // Trigger a re-render when receiving new props (when redux has more data).
+    this.updateDataSource(nextProps);
+  }
+
+  updateDataSource(props) {
+    // See the ListView.DataSource documentation for more information on
+    // how to properly structure your data depending on your use case.
+    let rows = props.listData.items;
+
+    let ids = rows.map((obj, index) => index);
+
+    this.setState({
+      dataSource: this.state.dataSource.cloneWithRows(rows, ids),
+    });
+  }
+
+  _rowHasChanged(r1, r2) {
+    // You might want to use a different comparison mechanism for performance.
+    return JSON.stringify(r1) !== JSON.stringify(r2);
+  }
+
+  _renderRefreshControl() {
+    // Reload all data
+    return (
+      <RefreshControl
+        refreshing={this.props.transactions.isFetching}
+        onRefresh={this._loadMoreContentAsync.bind(this)}
+      />
+    );
+  }
+
+  async _loadMoreContentAsync() {
+    // In this example, we're assuming cursor-based pagination, where any
+    // additional data can be accessed at this.props.listData.nextUrl.
+    //
+    // If nextUrl is set, that means there is more data. If nextUrl is unset,
+    // then there is no existing data, and you should fetch from scratch.
+    this.props.dispatch(fetchMoreContent(this.props.listData.nextUrl));
+  }
+
+  render() {
+    return (
+      <ListView
+        renderScrollComponent={props => <InfiniteScrollView {...props} />}
+        dataSource={this.state.dataSource}
+        renderRow={...}
+        refreshControl={this._renderRefreshControl()}
+        canLoadMore={!!this.props.listData.nextUrl}
+        onLoadMoreAsync={this._loadMoreContentAsync.bind(this)}
+      />
+    );
+  }
+}
+
+const mapStateToProps = (state) => {
+  return {listData: state.listData};
+};
+
+export default connect(mapStateToProps)(ExampleComponent);
+```
+
 ## Tips and Caveats
 
 - Horizontal scroll views are supported
 - When you load more content in an infinite ListView, the ListView by default will render only one row per frame. This means that for a short amount of time after loading new content, the user could still be very close to the bottom of the scroll view and may trigger a second load.
+- Known issue: Make sure your data reaches the bottom of the screen, otherwise scroll events won't trigger. See [exponentjs/react-native-infinite-scroll-view#9](https://github.com/exponentjs/react-native-infinite-scroll-view/issues/9) for more details.
```

Comment:
can you clarify it's just the initial data that needs to fill a screen, not each subsequent load?